torch/_export/converter.py

```diff
@@ -427,6 +429,28 @@ def convert_as_noop(self, node: torch._C.Node):
         output_name = node.output().debugName()
         self.name_to_node[output_name] = args[0]
 
+    def convert_profiler__record_function_enter_new(self, node: torch._C.Node):
```

Comment:
any reason why these two profiler::* ops cannot use the `convert_standard_operators`?